### PR TITLE
Dropdowns populating from JSONs

### PR DESCRIPTION
**Pull request by kevharvell**

### CODE REVIEW

File: src/components/MapForm.js [hunk 1]
```diff
@@ -211,15 +210,70 @@ export class MapForm extends Component {
...
       }
     }));
-    this.updateFields(lat, lng)
+    this.updateFields(lat, lng);
   }
 
   getAPIGraphData = (event) => {
     event.preventDefault()
-    fetch(`https://api.census.gov/data/timeseries/eits/${this.state.tableId}?get=cell_value,data_type_code,time_slot_id,error_data,category_code,seasonally_adj,geo_level_code&time=from+${this.state.fromYear}`)
+    fetch(`https://api.census.gov/data/timeseries/eits/${this.state.tableId}?get=cell_value,data_type_code,time_slot_id,error_data,category_code,seasonally_adj&time=from+${this.state.fromYear}`)
```
Comment (BartHough) [inline src/components/MapForm.js]: Looks like `geo_level_code` was dropped here, can we add that back in?

File: src/components/MapForm.js [hunk 2]
```diff
@@ -106,20 +106,18 @@ export class MapForm extends Component {
...
-    }
-    else {
-      dataTypes = dtcRessales;
-      categories = ccRessales;
-    }
+    
+    const data = await import(`../apiJsons/${tableId}.json`)
+
+    let dataTypes = this.extractCategoryDataDropdownOptions(data.default[tableId][DATA_TYPE_CODE].data_type_code);
+    let categories = this.extractCategoryDataDropdownOptions(data.default[tableId][CATEGORY_CODE].category_code);
```
Comment (BartHough) [inline src/components/MapForm.js:116]: Very clean, nice!

**Review by BartHough — CHANGES_REQUESTED** [review]
Looks good once we add `geo_level_code` back to the query

File: src/components/MapForm.js (hunk 1 above)
Comment (kevharvell) [reply src/components/MapForm.js]: oops, added it back, thanks

**Review by BartHough — APPROVED** [review]
Nice work!

**Review by DRot88 — APPROVED** [review]

**Pull request merged**